cloudinit/util.py

```diff
@@ -986,28 +986,21 @@ def load_yaml(blob, default=None, allowed=(dict,)):
 
 
 def read_seeded(base="", ext="", timeout=5, retries=10, file_retries=0):
-    if base.startswith("/"):
-        base = "file://%s" % base
-
-    # default retries for file is 0. for network is 10
```

Comment:
Why was this logic dropped?

Reply:
This was dropped, if I remember right, because read_file_or_url does exactly the same thing, so doing it here also was redundant.

Reply:
Correct read_file_or_url already does this.